db/seeds.rb

```diff
@@ -5,3 +5,17 @@
 #
 #   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
 #   Mayor.create(name: 'Emanuel', city: cities.first)
+
+category1 = Category.create(name: 'Coffee')
+category2 = Category.create(name: 'Brewing')
+category3 = Category.create(name: 'Merchandise')
+
+item1 = category1.items.create(title: 'Non-fat latte with caramel drizzle', description: 'For the skinny but caffeinated', price: 1000, photo: 'http://i.huffpost.com/gen/1270227/thumbs/o-COFFEE-570.jpg?5')
+item2 = category1.items.create(title: 'Grande iced sugar-free vanilla latte with soy milk', description: 'For the healthy organic individual', price: 800, photo: 'http://i.huffpost.com/gen/1269829/thumbs/o-COFFEE-570.jpg?1')
```

Comment:
Useless assignment to variable - `item2`.<br>Line is too long. [224/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.